src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).orElseThrow(() -> new DataProcessingException(
+                "Can't get cinema hall by id" + id));
+    }
```

Comment:
DataProcessingException I think it's for DAO layer, in service layer we need another type of exception, but it's up to you, I don't sure
```suggestion
        return cinemaHallDao.get(id).orElseThrow(() -> new NoSuchElementException(
                "Can't get cinema hall by id" + id));
    }
```